### DISCUSSION TITLE
Windows: Docker: Standalone CLI --watch not rebuilding on file changes

### DISCUSSION THREAD

**Discussion by jpau-besmartee** [tailwindlabs/tailwindcss#8100] [Help]
**What version of Tailwind CSS are you using?**

3.0.15

**What build tool (or framework if it abstracts the build tool) are you using?**

[tailwindcss-rails](https://github.com/rails/tailwindcss-rails)

**What operating system are you using?**

Windows with app built in Docker on an Alpine system

**Describe your issue:**

I have a rails application which makes use of the tailwindcss [standalone CLI](https://tailwindcss.com/blog/standalone-cli). Like I mentioned above this application is run on an Alpine docker container and uses docker compose where we mount the volumes like so:
```
volumes:
  - .:/app:cached
```

Currently everything works as expected including the rails application itself and running the tailwind build process. The rails gem used to run the tailwind executable runs the following command on the container when I want to build my tailwind file:
```
 /usr/local/bundle/gems/tailwindcss-rails-2.0.5-x86_64-linux/exe/x86_64-linux/tailwindcss -i /app/app/assets/stylesheets/application.tailwind.css -o /app/app/assets/builds/tailwind.css -c /app/config/tailwind.config.js
```

This runs and successfully builds the tailwind.css file in the correct location. Now when I run the rails gem to have it watch, it runs the following command (basically the same thing but with the `-w` option):
```
/usr/local/bundle/gems/tailwindcss-rails-2.0.5-x86_64-linux/exe/x86_64-linux/tailwindcss -i /app/app/assets/stylesheets/application.tailwind.css -o /app/app/assets/builds/tailwind.css -c /app/config/tailwind.config.js -w
```

When that command runs it says:
```
Rebuilding...
Done in xxx ms.
```

But any time I make changes to the files located within my tailwind.config.js `content` option nothing happens. There was this [issue](https://github.com/tailwindlabs/tailwindcss/issues/6343) where it seemed someone was facing a similar issue but not when using the standalone CLI so I'm not sure if this is still an issue in this case.

I should mention that both building and watching initially build the tailwind.css file as expected and the changes appear in the application so there doesn't appear to be any issue reading the files in the tailwind.config.js under the `content` options.

I have left out a lot of potential information and can add it as requested since I'm not sure how much would be useful for understanding the problem/troubleshooting.

**Comment by improbablistic** [comment]
Experienced the same issue when running tailwind standalone on watch mode in debian docker container in windows host. If polling is supported in your environment, using `--watch --poll` flags should be a viable back up solution.
